### PR TITLE
Addition of a new HWS.

### PR DESCRIPTION
**Pull request by ghairapetian**

### CODE REVIEW

**Review by cstirdivant — CHANGES_REQUESTED** [review]

File: ontology/yaml/resources/HVAC/entity_types/HWS.yaml [hunk 1]
```diff
@@ -313,6 +313,12 @@ HWS_SS_HWTTC_PWDPM2X:
   opt_uses:
   - compressor_run_status
 
+HWS_HWSWC:
+  description: Supply water temperature control with valve control
+  implements:
+  - HWS
+  - HWSWC
```
[the remark this answers is not in the document]
Comment (ghairapetian) [reply ontology/yaml/resources/HVAC/entity_types/HWS.yaml:320]: Hello, 

This valve is neither a bypass valve or an isolation valve. It's a control valve -- it mixed the return water with the heat water from the production system to control the supply water temperature based on a setpoint.

[the remark this answers is not in the document]
Comment (ghairapetian) [reply ontology/yaml/resources/HVAC/entity_types/HWS.yaml:320]: image

**Comment by cstirdivant** [comment]
Hi @ghairapetian thank you for the additional information, based on this, it is most-correct to model the valve as a mixing valve. See the abstract type `MXVPM` for an example. (If you have both the valve percentage command and percentage sensor, please add the sensor as opt_uses).

File: ontology/yaml/resources/HVAC/entity_types/HWS.yaml [hunk 2]
```diff
@@ -313,6 +313,13 @@ HWS_SS_HWTTC_PWDPM2X:
   opt_uses:
   - compressor_run_status
 
+HWS_SWTC_HWVM:
+  description: Supply water temperature control with valve control
+  implements:
+  - HWS
+  - SWTC
+  - HWVM
```
Comment (cstirdivant) [inline ontology/yaml/resources/HVAC/entity_types/HWS.yaml:321]: Change to `MXVPM` and add percentage sensor as opt_uses on the `MXVPM` type.

Comment (ghairapetian) [reply ontology/yaml/resources/HVAC/entity_types/HWS.yaml]: Correction done.

**Review by cstirdivant — APPROVED** [review]

**Pull request merged**